drop/demo/config_relative.yaml

```
@@ -8,6 +8,7 @@ geneAnnotation:
     v29: Data/gencode_annotation_trunc.gtf
 genomeAssembly: hg19
 hpoFile: null
+random_seed: true  # just for demo data, remove for proper analysis
```

This could be used to have a fixed seed. This is also helpful if you want to reproduce an error. Currently, it is misleading. 

```suggestion
random_seed: true  # use this to fully gain reproducible data in debug mode
```